test/04-verifying/35-diffheaders.js

```diff
@@ -152,3 +143,41 @@ test('test diffheaders return values adding From header field', async t => {
   }
 });
 
+test('Memory Leak Test For DiffHeaders Variable', async t => {
+  try {
+    return iterate.async(async () => {
+      var opendkim = new OpenDKIM();
+      opendkim.query_method('DKIM_QUERY_FILE');
+      opendkim.query_info('../fixtures/testkeys');
+      await opendkim.verify({id: undefined});
+      var header = 'From: Murray S. Kucherawy <yo@mismo.com>';
+      await opendkim.header({
+        header: header,
+        length: header.length
+      });
+      for (var i = 0; i < 3; i++) {
```

Comment:
Just 3 headers more to have a relatively big diffheaders array. 
Grows very quickly, so i've tested with no more than 5 added headers. 
For more added headers the test take too long for my tiny linux box but no memory growing issues.